### PR TITLE
Contract test commands for ioctl

### PR DESCRIPTION
**Pull request by Frankonly**

### CODE REVIEW

File: ioctl/cmd/contract/contracttestbytecode.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	"github.com/iotexproject/iotex-core/ioctl/cmd/action"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+)
+
+// Multi-language support
+var (
+	testBytecodeCmdUses = map[config.Language]string{
+		config.English: "bytecode (CONTRACT_ADDRESS|ALIAS) PACKED_ARGUMENTS [AMOUNT_IOTX]",
```
Comment (dustinxie) [inline ioctl/cmd/contract/contracttestbytecode.go:25]: why this need IOTX amount?

File: ioctl/cmd/contract/contracttestbytecode.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+	amount := big.NewInt(0)
+	if len(args) == 3 {
+		amount, err = util.StringToRau(args[2], util.IotxDecimalNum)
+		if err != nil {
+			return output.NewError(output.ConvertError, "invalid amount", err)
+		}
+	}
+
+	result, err := action.Read(contract, amount, bytecode)
```
Comment (dustinxie) [inline ioctl/cmd/contract/contracttestbytecode.go:70]: action read cmd also calls Read(contract, bytecode)

what is the diff from action read?

File: ioctl/cmd/contract/contracttestfunction.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+	"github.com/iotexproject/iotex-core/ioctl/cmd/action"
+	"github.com/iotexproject/iotex-core/ioctl/config"
+	"github.com/iotexproject/iotex-core/ioctl/output"
+	"github.com/iotexproject/iotex-core/ioctl/util"
+)
+
+// Multi-language support
+var (
+	testFunctionCmdUses = map[config.Language]string{
+		config.English: "function (CONTRACT_ADDRESS|ALIAS) ABI_PATH FUNCTION_NAME [AMOUNT_IOTX] " +
```
Comment (dustinxie) [inline ioctl/cmd/contract/contracttestfunction.go:25]: how is this diff from contractinvokefunction.go?

Comment (dustinxie) [reply ioctl/cmd/contract/contracttestfunction.go:25]: maybe i miss something, are you adding new cmds in this PR?
looks like it is same as existing contractinvokefunction.go

File: ioctl/cmd/contract/contracttestbytecode.go (hunk 1 above)
Comment (Frankonly) [reply ioctl/cmd/contract/contracttestbytecode.go:25]: Not real token, but simulating amount

File: ioctl/cmd/contract/contracttestbytecode.go (hunk 2 above)
Comment (Frankonly) [reply ioctl/cmd/contract/contracttestbytecode.go:70]: Contract cmds are independent cmd cluster. We will have more cmds in contract test.

File: ioctl/cmd/contract/contracttestfunction.go (hunk 3 above)
Comment (Frankonly) [reply ioctl/cmd/contract/contracttestfunction.go:25]: It use Read() rather than Execute()

**Pull request merged**